casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
```

Comment:
If it's relevant there are also ~20,000,000 incoming messages (vast majority of messages are not replied to)